src/test/kotlin/uk/gov/justice/digital/hmpps/deliusapi/service/contact/UpdateContactTest.kt

```diff
@@ -268,9 +278,13 @@ class UpdateContactTest : ContactServiceTestBase() {
       .thenReturn(if (having) provider else null)
   }
 
-  private fun havingOutcomeType(having: Boolean = true) {
+  private fun havingOutcomeType(having: Boolean = true, isActive: Boolean = true,) {
     val mock = whenever(validationService.validateOutcomeType(request, contact.type))
-    if (having) mock.thenReturn(outcome) else mock.thenThrow(BadRequestException("bad outcome"))
+    if (isActive) {
+      if (having) mock.thenReturn(outcome) else mock.thenThrow(BadRequestException("bad outcome"))
+    } else {
+      mock.thenCallRealMethod()
+    }
```

Comment:
Same here, would it be better to mock an exception?
```suggestion
    if (!having) {
      mock.thenThrow(BadRequestException("bad outcome"))
    } else if (!isActive) {
      mock.thenThrow(BadRequestException("inactive outcome"))
    } else {
      mock.thenReturn(outcome)
    }
```